src/ui/components/forms/VPNTextField.qml

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.0
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+import Mozilla.VPN 1.0
+import "../../themes/themes.js" as Theme
+import "./../../components"
+
+TextField {
```

Comment:
Can you confirm the input is selectable with the mouse?

Reply:
Confirmed on Mac